packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -3,32 +3,20 @@
 // HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
 // and then running this script with:
 // EMERGENCY_EXECUTOR=<EMERGENCY-EXECUTOR-ADDRESS> \
-// EMERGENCY_QUORUM=<EMERGENCY-QUORUM> \ # Decimal value between 5M and 10M
+// UPGRADER_WALLET=<UPGRADER-WALLET> \ # Address of the wallet that will be used to upgrade the contracts.
```

Comment:
Please also update README on removing EMERGENCY_QUORUM and adding UPGRADER_WALLET